ec2-backup.sh

```diff
@@ -44,9 +49,36 @@ createVolume() {
                
 
         fi
-
+	
+	##If volume flag value is empty we create a ne wone and attach
+	if [$v == ' ' ]; then
+		createVolume = aws ec2 create-volume --size $CHECK --availability-zone $timeZone --volume-type standard
+		volumeId = aws ec2 describe-volumes | grep VolumeId | head -1 | awk '{print $2}' | sed 's/\"//g' | sed 's/\,//g'
+		attachVolume = aws ec2 attach-volume --volume-id $volumeId --instance-id $instanceId --device /dev/sdf
+		mountVolume = ssh -i ec2BackUpKeyPair.pem ec2-user@$publicDns 'sudo file -s /dev/sdf | sudo mkfs -t ext4 /dev/sdf | sudo mkdir /data | sudo mount /dev/sdf /data'
+	
+	##If volumen flag has a value, check if it is already attached. If so, echo an error and if not use that volume id to attach and mount
+	else
+		volumeState = aws ec2 describe-volumes | grep State | head -1 | awk '{print $2}' | sed 's/\"//g' | sed 's/\,//g'
+		if [ volumeState == 'attached' ]; then
+			echo "Please specify a volume that is available."
```

Comment:
@rfarwaha , @skarinat  Feel free to add a better error message here. It's just a placeholder for now.